CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ internal API changes are not present.
 Main (unreleased)
 -----------------
 
+v0.36.0 (2023-08-29)
```

Comment:
IIRC, the VERSION header should include the rc number, in case we have to release more than one, right?

Reply:
☝️ what Paul said